.buildkite/release.yml

```diff
@@ -1,3 +1,7 @@
 steps:
-  - label: ":wave: Greetings"
-    command: "echo 'My release pipeline!'"
+  - label: ":rocket: Release"
+    command: .buildkite/release.sh
+    agents:
+      image: "docker.io/library/golang:1.21"
+      cpu: "8" # The release flow needs that many or hits https://github.com/goreleaser/goreleaser/issues/304
```

Comment:
Is this the right issue number? `Add config for number of parallel builds #304`?